sda-commons-metadata-context/build.gradle

```diff
@@ -0,0 +1,16 @@
+dependencies {
+  api 'org.springframework.cloud:spring-cloud-starter-openfeign'
+  api 'org.springframework.boot:spring-boot-starter-web'
```

Comment:
That means that all modules that depend on "sda-commons-metadata-context" will still get all the web dependencies. Have we achieved the goal of the refactoring? Would probably result in more modules if we want to separate "metadata-context-core" + "metadata-context-web". What do we prefer?

Reply:
Resolved on https://github.com/SDA-SE/sda-spring-boot-commons/pull/211